praw/const.py

```diff
@@ -91,14 +91,17 @@
     'my_subreddits':          'subreddits/mine/subscriber/',
     'marknsfw':               'api/marknsfw/',
     'modmail_archive':        'api/mod/conversations/{id}/archive',
+    'modmail_bulk_read':      'api/mod/conversations/bulk/read',
```

Comment:
I haven't pushed on this much, but now that you're pushing a lot more code, consider focusing on making your commits more atomic. Many of your commits are split in a horizontal way, that is, they focus on the area of code they touch (e.g., paths, tests, implementation). One aspect of atomic commits is to split things vertically. That is focus on one bit of value that is completely added (paths, implementation, and tests), and separate commits based on those micro-features.

Breaking commits in this way makes it easier to revert a single commit to remove the entirety of a small feature. For example, if Reddit were to remove the bulk_read feature, I would have to manually delete the relevant parts of the code, along with the associated tests. However, if these commits were split vertically, then I might be able to revert a single commit and still have all the tests pass.

Feel free to ask questions if you're not sure. This information is mostly for your benefit. I'm happy to have you merge the code in as-is since I see little value in ensuring atomic commits. If you aren't already aware, `git add -p` is your friend for helping construct such commits.